webservices/src/main/java/com/javaschool/webservices/configuration/RabbitMQConfig.java

```diff
@@ -0,0 +1,53 @@
+package com.javaschool.webservices.configuration;
+
+import org.springframework.amqp.core.Binding;
+import org.springframework.amqp.core.BindingBuilder;
+import org.springframework.amqp.core.Exchange;
+import org.springframework.amqp.core.ExchangeBuilder;
+import org.springframework.amqp.core.Queue;
+import org.springframework.amqp.core.QueueBuilder;
+import org.springframework.amqp.core.TopicExchange;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@Configuration
+public class RabbitMQConfig {
+
+	private final String SHIPPING_QUEUE_NAME;
```

Comment:
this are not really constants, just final variables, you should follow the camelCase convention.

Reply:
now using a properties configuration class I follow the camelCase convention.